ansible/roles/docker/templates/daemon.json.j2

```diff
@@ -0,0 +1,3 @@
+{
+    "iptables": false
```

Comment:
Do we need the `"experimental": true` here?

Reply:
No, it was just copied in from my `jenkins-up` repo; https://github.com/mblomdahl/jenkins-up/blob/master/roles/docker/tasks/configuration.yml

This used to be required in order to **build** the multi-image containers. It has no place in production environments!